README.md

```diff
@@ -37,6 +37,6 @@ This project requires Ruby 2.7.4.
 1. [Extensions](./doc/extensions.md)
 1. [Evaluation](./doc/evaluation.md)
 
-## Schema
+## Database Diagram
```

Comment:
Is there a reason the readme was changed?

Reply:
Wasn't me, was @ruezheng if you look at the commit. Not sure why it comes up in my pull request. Looks like the database diagram was just added in.